### DISCUSSION TITLE
AG 1.2.0 Beta6 + Crossover 21 + Monterey + Logic

### DISCUSSION THREAD

**Discussion by CrossGridder** [apohl79/audiogridder#797] [Help]
I was able to have everything work except for the fact that the UI must be the generic one. If I try to get the plugin UI, the AG server inside Crossover restarts and it enters a loop for binding port. Reading here and there it seems that it happened to someone else and the solution would be a better sandbox handling (correct me if I am wrong). However, this solution is so fast and light on the cpu that there should be a way to see the UI.

**Comment by JamesXXV** [comment]
did you install directx for modern games, and the c++ visual studio redistributables, core fonts and xml parser?(not sure what xml parser would be helpful with but its recommend to run gta v), yeah I know a game isn't your focus right now but if its having trouble with the graphics maybe a guide for a game on crossover will help you, since they use similar renders and all , example softube uses OpenGL on all plugs.

**Reply by CrossGridder** [reply]
Thanks for your reply. Could you point me out to any game guide that you think I could adapt to what I am trying to achieve? Thanks

Edit 1: When trying to install microsoft frame network 4 it says it is already installed. For directx, crossover 21 supports directx11 natively.

**Reply by JamesXXV** [reply]
yeah so.... I couldn't get the graphics up either even with directx installed so I personally don't think it will work with crossover but if you need a guide I think this may help, https://www.youtube.com/watch?v=HBnJmp58hn8 , https://www.applegamingwiki.com/wiki/CrossOver

**Reply by CrossGridder** [reply]
I am at a point where after lots of trials and erros, I am able to see the gui for a second when restarting the server, but then it disappears. I guess it is because I disabled screen capture and local, it appears for that moment on the windows side. So, on the crossover side the guide is rendered, the problem is when it is transferred to the plugin.

**Reply by CrossGridder** [reply]
I've started this thread in order to see if there is any possibility to implement it in a future version of AG server (or plugin? Andreas for sure will find the best solution in case it could be possible) Like the idea if you really like it (personally, I think it could be groundbreaking), maybe it could enter the pipeline soon if you show some love :)

Edit: removed wrong link

**Reply by JamesXXV** [reply]
I tried going to the discussion it says error 404 page not found. What'd you find with your trial and error? Does it even work now? I tried and the server just kept restarting on me.

**Comment by JamesXXV** [comment]
I tried all the steps you mentioned but surprisingly it just says timeout error when loading the plugin and crashes the DAW.
